packages/indy-vdr/src/dids/__tests__/IndyVdrSovDidResolver.test.ts

```diff
@@ -0,0 +1,127 @@
+import { JsonTransformer } from '@aries-framework/core'
+
+import didSovR1xKJw17sUoXhejEpugMYJFixture from '../../../../core/src/modules/dids/__tests__/__fixtures__/didSovR1xKJw17sUoXhejEpugMYJ.json'
+import didSovWJz9mHyW9BZksioQnRsrAoFixture from '../../../../core/src/modules/dids/__tests__/__fixtures__/didSovWJz9mHyW9BZksioQnRsrAo.json'
```

Comment:
I think my preference would go to duplicating the fixtures, mainly because the different packages may diverge over time and it becomes easier I think when all packages are more 'isolated'.

Reply:
Yeah I was hesitating about that. I copied them, as it will make more sense once did:indy resolver is added to the package (as there are more specific fixtures to add).